lib/src/strings.dart

```diff
@@ -13,3 +13,23 @@ bool isNotBlank(String? v) => isFalse(isBlank(v));
 /// `since 0.0.1`
 // TODO improver performance by checking if every character is whitespace
 bool isBlank(String? v) => isNull(v) || v!.trim().isEmpty;
+
+/// Returns `true` if [left] is equal to [right] irrespective of case
+///
+/// `since 0.1.0`
+bool equalsIgnoreCase(String? left, String? right) {
+  return left?.toLowerCase() == right?.toLowerCase();
```

Comment:
FYI this may not work in other locales. But I am guessing it is good enough for now

Reply:
Yep. Other libraries typically offer a different function or argument for local agnostic comparison. Not a requirement right now.